exporter/doc/addresses.go

```diff
@@ -0,0 +1,49 @@
+package doc
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	"github.com/projectcypress/cdatools/models"
+)
+
+// Addresses come in groups, so it should be useful to print them as a group,
+// rather than one at a time
+type Addresses struct {
+	Addresses []models.Address
+}
```

Comment:
I mentioned in another comment, but these structs do more than just holding a slice. They are representatives in the `doc` package of new template documents that we're printing out instead of partial templates that are recompiled at runtime.

In this case, the template represents multiple addresses. So the document is called `Addresses`.